sdk/communication/azure-communication-identity/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "java",
   "TagPrefix": "java/communication/azure-communication-identity",
-  "Tag": "java/communication/azure-communication-identity_5214b2a09e"
+  "Tag": "java/communication/azure-communication-identity_b94837dbf2"
```

Comment:
have you already updated the recordings?

Reply:
Yes, I have to as there are new test written and CI build takes recordings from asset repo